Dockerfile

```diff
@@ -72,19 +72,19 @@ RUN apt-get update -qq \
 # Install miniconda2
 # still need python 2 for gradunwarp
 ENV PATH="/usr/local/miniconda/bin:$PATH"
-RUN curl -fsSLO https://repo.continuum.io/miniconda/Miniconda2-4.5.4-Linux-x86_64.sh && \
-    bash Miniconda2-4.5.4-Linux-x86_64.sh -b -p /usr/local/miniconda && \
-    rm Miniconda2-4.5.4-Linux-x86_64.sh && \
+RUN curl -fsSLO https://repo.continuum.io/miniconda/Miniconda2-4.7.12.1-Linux-x86_64.sh && \
+    bash Miniconda2-4.7.12.1-Linux-x86_64.sh -b -p /usr/local/miniconda && \
+    rm Miniconda2-4.7.12.1-Linux-x86_64.sh && \
     conda config --add channels conda-forge && \
     conda install -y mkl=2019.3 mkl-service=2.0.2 numpy=1.16.4 nibabel=2.4.1 pandas=0.24.2 && sync && \
     conda clean -tipsy && sync && \
-    pip install --no-cache-dir pybids==0.9.1
+    /usr/local/miniconda/bin/pip install --no-cache-dir pybids==0.9.1
 
 # Install connectome-workbench
 WORKDIR /opt
 RUN apt-get -qq update && \
-    apt-get install -yq libfreetype6 libglib2.0 && \
-    wget -q https://ftp.humanconnectome.org/workbench/workbench-linux64-v1.3.2.zip -O wb.zip \
+    apt-get install -yq libfreetype6 libglib2.0
+RUN wget -q https://www.humanconnectome.org/storage/app/media/workbench/workbench-linux64-v1.4.1.zip -O wb.zip \
```

Comment:
Freetype and glib are specifically needed for workbench, so I think it makes sense to keep `apt-get` and the workbench install in the same stage of the docker build.